index.js

```diff
@@ -67,11 +67,18 @@ class ServerlessPythonRequirements {
       }
       if (this.custom().dockerizePip) {
         cmd = 'docker';
+
+        let image = `lambci/lambda:build-${runtime}`
```

Comment:
I don't have any issues. I'm not much of a JS developer, so whatever you think is best. I'll update the PR in a minute.